salt/kube-apiserver/apiserver.jinja

```diff
@@ -49,6 +61,8 @@ KUBE_API_ARGS="--advertise-address={{ salt.caasp_net.get_primary_ip() }} \
                --tls-cert-file={{ pillar['ssl']['kube_apiserver_crt'] }} \
                --tls-private-key-file={{ pillar['ssl']['kube_apiserver_key'] }} \
                --tls-ca-file={{ pillar['ssl']['ca_file'] }} \
+               --proxy-client-cert-file={{ pillar['ssl']['kube_apiserver_proxy_client_crt'] }} \
```

Comment:
Shouldn't we enable/disable this feature with a pillar?

Reply:
It can be turned on all the time. It's being used only if the user defines a new validation service by creating the right kubernetes objects via `kubectl`